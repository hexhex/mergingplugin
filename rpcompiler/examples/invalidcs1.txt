[common signature]
predicate:x/0;
pred _ this is a syntax error _ icate:y/0;

[belief base]
name:bb1;
mapping: "x. y.";

[revision plan]
{
	bb1
}
