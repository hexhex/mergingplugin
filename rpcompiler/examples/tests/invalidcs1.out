Error: Character '_' in file ../../rpcompiler/examples/tests/../invalidcs1.txt, line 3 is an invalid token

Error: Character '_' in file ../../rpcompiler/examples/tests/../invalidcs1.txt, line 3 is an invalid token

Error in ../../rpcompiler/examples/tests/../invalidcs1.txt, line 3: predicate definition expected
Parsing finished with errors:
   3 errors, 0 warnings
